### PR TITLE
LPD-18648 Delete notification if it is no longer relevant

### PR DESCRIPTION
**Pull request by brookedalton**
https://liferay.atlassian.net/browse/LPD-18648

### CODE REVIEW

**Comment by brookedalton** [comment]
https://github.com/liferay-publications/liferay-portal/assets/46908843/2dc57398-e490-4802-887d-b1da9305aaf3

**Comment by pei-jung** [comment]
@brookedalton can you add a test for this? You'll need to rebase on top of https://github.com/liferay-publications/liferay-portal/pull/1062 in order to add a new case since I refactored the test class quite a bit.

**Comment by brookedalton** [comment]
ci:test:sf

**Comment by brookedalton** [comment]
ci:test:sf

File: modules/apps/change-tracking/change-tracking-test/src/testIntegration/java/com/liferay/change/tracking/internal/test/PublicationUserNotificationHandlerTest.java [hunk 1]
```diff
@@ -81,6 +81,26 @@ public void setUp() throws Exception {
 		_group = GroupTestUtil.addGroup();
 	}
 
+	@Test
+	public void testDeletedBodyNoLongerApplies() throws Exception {
```
Comment (pei-jung) [inline modules/apps/change-tracking/change-tracking-test/src/testIntegration/java/com/liferay/change/tracking/internal/test/PublicationUserNotificationHandlerTest.java:85]: Rename this to `testGetBodyForDeletedPublication`.

**Review by pei-jung — CHANGES_REQUESTED** [review]

**Comment by brookedalton** [comment]
ci:test:sf

**Review by pei-jung — APPROVED** [review]

**Comment by pei-jung** [comment]
ci:test:relevant

**Comment by pei-jung** [comment]
ci:test:publications

**Comment by pei-jung** [comment]
ci:forward

**Pull request closed without merging**